tests/unittest/_torch/speculative/test_force_accepted_tokens.py

```
@@ -0,0 +1,365 @@
+        static_output.copy_(static_input)
+
+    graph_outputs = []
+    for _ in range(n_iters):
+        # Reset input each replay so the override semantics match eager
+        # (which also starts from all-1s every iteration).
+        static_input.fill_(1)
+        graph.replay()
+        torch.cuda.synchronize()
+        graph_outputs.append(static_output.clone())
+
+    for i, (eager, graphed) in enumerate(zip(eager_outputs, graph_outputs)):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)